resources/sass/layout/_header.scss

```diff
@@ -0,0 +1,157 @@
+header {
+    width: 100%;
+    nav {
+        -webkit-box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
```

Comment:
if you are using postcss, do you need these ?

Reply:
No